01-hello-world/helloWorld.js

```diff
@@ -3,8 +3,8 @@
 //----------------------------------------------------------------------------------------------------------------------
 
 function sayHello(name)
-{
-    // Code goes here
+{ 
+    console.log('Hello,', name + '!');
```

Comment:
This can be much more elegantly written if we use a template string. So, it could be rewritten as:

```javascript
function sayHello(name)
{
    console.log(`Hello, ${ name }!`);
}
```